.gitignore

```diff
@@ -105,3 +105,6 @@ deps/build
 # CCache within Docker
 /.docker_ccache/
 /.docker_images/
+
+# Pre-commit stuff
```

Comment:
I don't really think this comment is really adding anything. I'd probably prefer just not having a comment here than having this comment. A better comment might be `# git hooks`.

Reply:
That's better! Will do.